<p><b>[TRACE]</b>&nbsp;Arguments: [ 'Test Data >>> Username: locked_out_user, Password: secret_sauce, Error: Epic sadface: Sorry, this user has been locked out.' | 'Login with locked out user' ]</p><p><b>[INFO]</b>&nbsp;07-21-2025 10:15:04 AM - INFO - root : Test Data >>> Username: locked_out_user, Password: secret_sauce, Error: Epic sadface: Sorry, this user has been locked out.</p><p><b>[TRACE]</b>&nbsp;Return: None</p>